maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -397,6 +397,8 @@ class Maestro(private val driver: Driver,
     }
 
     fun inputText(text: String) {
+        LOGGER.info("Inputting text: $text")
```

Comment:
Is this change intended?

Reply:
Yes, for some reason we did not have a log message for that command